content/en/docs/reference/kubectl/resource_printing/queries_and_options.md

```diff
@@ -0,0 +1,124 @@
+---
+title: "Queries and Options"
+linkTitle: "Queries and Options"
+weight: 5
+type: docs
+description: >
+    Prints result of queries
+---
+
+
+{{< alert color="success" title="TL;DR" >}}
+- Queries for Getting or Describing Resources
+{{< /alert >}}
+
+# Matching Objects from Get and Describing
+
+## Motivation
```

Comment:
Thanks.
We've realized that a straight conversion isn't right. This proposed tweak should help a lot though:
```suggestion
```

Reply:
Done!